File: Service_eHospital_DongNai_A_NSTL/Implement/NS_DM_TinhTrangQuanHeService.cs
namespace Service_eHospital_DongNai_A_NSTL.Implement
{
    public class NS_DM_TinhTrangQuanHeService : BaseService<NS_DM_TinhTrangQuanHe, INS_DM_TinhTrangQuanHeRepository>
    , INS_DM_TinhTrangQuanHeService
    {
    private readonly INS_DM_TinhTrangQuanHeRepository _NS_DM_TinhTrangQuanHeRepository;
    public NS_DM_TinhTrangQuanHeService(INS_DM_TinhTrangQuanHeRepository NS_DM_TinhTrangQuanHeRepository) : base(NS_DM_TinhTrangQuanHeRepository)
    {
    _NS_DM_TinhTrangQuanHeRepository = NS_DM_TinhTrangQuanHeRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A_NSTL/NS_DM_TinhTrangQuanHe.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { NS_DM_TinhTrangQuanHe } from './NS_DM_TinhTrangQuanHe.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class NS_DM_TinhTrangQuanHeService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'TinhTrangQuanHe_Id', 'MaTinhTrang', 'TenTinhTrang', 'TamNgung'];

    List: NS_DM_TinhTrangQuanHe[] | undefined;
    ListFilter: NS_DM_TinhTrangQuanHe[] | undefined;
    FormData!: NS_DM_TinhTrangQuanHe;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "NS_DM_TinhTrangQuanHe";
    }
}

